.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,33 @@
+Thank you for contributing to LangGraph!
```

Comment:
Anything to reduce the amount of copy would be great, otherwise folks will not read it